tests/runmode_test/test.yaml

```diff
@@ -0,0 +1,10 @@
+requires:
+  min-version: 7.0.0
+  pcap: false
+
+checks:
+  - shell:
+      args: suricata
```

Comment:
Since the Suricata verify suite will run Suricata, you don't have to invoke shell to run it.

Instead, the suricata-verify way would be to have a line before the checks where you would pass the args for Suricata to run with. Check for instance https://github.com/OISF/suricata-verify/blob/master/tests/config-includes-array/test.yaml#L6-L7, these means that Suricata-verify will run Suricata as
`suricata --dump-config -c suricata.yaml`.

The `-c suricata.yaml` part is added by suricata-verify, which will use a suricata.yaml file present in the added test directory, if you provide one.

Reply:
Okay, thank you for the feedback